src/msg/async/AsyncMessenger.cc

```diff
@@ -194,15 +194,23 @@ void Processor::accept()
 	} else if (r == -EMFILE || r == -ENFILE) {
 	  lderr(msgr->cct) << __func__ << " open file descriptions limit reached sd = " << listen_socket.fd()
 			   << " errno " << r << " " << cpp_strerror(r) << dendl;
-	  break;
+	  if (++accept_error_num > msgr->cct->_conf->ms_accept_error_nums) {
```

Comment:
where is `accept_error_num defined ?

Reply:
I have missed the AsyncMessenger.h . Fixed it.

Reply:
I have checked that ceph_abort() using in lots of code paths like:

src/osd/OSD.cc

	if (!store->exists(service.meta_ch, oid)) {
	  derr << __func__ << " missing pg_pool_t for deleted pool "
	       << pgid.pool() << " for pg " << pgid
	       << "; please downgrade to luminous and allow "
	       << "pg deletion to complete before upgrading" << dendl;
	  ceph_abort();
	}
      }
    }
  }